frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -244,6 +365,21 @@ export default ({
       this.$refs.textarea.value = this.$refs.textarea.value + emoticon.native
       this.closeEmoticon()
       this.updateTextWithLines()
+    },
+    startMentioning (keyword, position) {
+      this.ephemeral.mentioning.options = this.users.concat([{
+        // TODO: use group picture here or broadcast icon
+        username: 'here', displayName: 'here', picture: ''
```

Comment:
Don't forget this also needs to be updated

Reply:
And maybe it would be better if the possibilities were all retrieved from the same place to DRY things, so that you don't have to update multiple sections of code if things are changed